src/ra_machine.erl

```diff
@@ -242,8 +241,6 @@
 
 -callback tick(TimeMs :: milliseconds(), state()) -> effects().
 
--callback snapshot_installed(ra_snapshot:meta(), state()) -> effects().
-
 -callback snapshot_installed(Meta, OldMacVer, OldState, NewState) -> Effects
```

Comment:
Looks like this needs an update to be `State, Meta, OldState, OldMeta`

Reply:
oops yea